src/lib/components/molecules/waffled-squares/index.jsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import { SvgSquare } from '$particles/svg-square'
+
+export const WaffledSquare = ({
+  name = 'Germany',
+  squaresTotal = 96,
+  waffleWidth = 100,
+  waffleHeight = waffleWidth,
```

Comment:
Rather than setting the width and height of the component explicitly, I think we should try and build components in a way that they scale to the size of their container. 

The reason for that is that they become much easier to use in responsive designs, where you might not know the explicit pixel width and height until runtime.